dasea/miners/clojars.py

```diff
@@ -15,7 +15,7 @@
 CLOJARS_VERSION_URL = "https://clojars.org/{pkg_name}/versions/{version}"
 
 HEADERS = {
-    "User-Agent": "DaSEA Research Project (Please don't ban, daseaITU@gmail.com)",
+    "User-Agent": "DaSEA Research Project (Please don't ban, https://dasea.org)",
```

Comment:
Why domain?

Reply:
I thought to change it, so they can see more about the project as the email is still in the From headers

Reply:
Makes sense - oki doki